### ISSUE TITLE
Mobile query example is broken

### ISSUE THREAD

**Issue by tsauerwein** [camptocamp/ngeo#982]
... in branch 2.0. Is there a fix that should be backported?

**Comment by fredj** [comment]
Maybe #692 ?

**Comment by tsauerwein** [comment]
I tried to apply the two changes, but it's still broken:

```
TypeError: Cannot read property 'split' of undefined
    at ngeo.Query.addSource 
```

**Comment by tsauerwein** [comment]
And I am also seeing this error when opening the mobile app of 2.0.

**Comment by fredj** [comment]
Humm, this issue should have been corrected with #980

**Comment by tsauerwein** [comment]
> fredj merged commit ...  2 hours ago

I updated my local check-out, now it works. Thanks, Fred!
